static/src/stylesheets/module/facia-garnett/_container.scss

```diff
@@ -453,6 +453,8 @@ $header-image-size-desktop: 100px;
     padding: 0;
     color: $brightness-46;
     text-align: right;
+    // min-width needed to ensure button is visible on Safari
```

Comment:
@zeftilldeath 😆 the big question is why  `min-width` is needed on Safari
@regiskuckaertz any idea?

Reply:
My assumption was Safari couldn't understand how wide the content was in the absolutely positioned div.

Reply:
Yes. An absolutely positioned element takes the dimensions of its children by default, so it may be they lack any (is there an SVG? Browsers don’t agree on how to deal with these very often).